src/components/ui/select/Select.tsx

```diff
@@ -0,0 +1,32 @@
+import type { FC } from 'react';
+import styles from './Select.module.css';
+
+interface Props {
+  setSelectedValue: (value: string | null) => void;
+  options: (number | string)[];
+  defaultValue?: string | number | null;
+}
+
+export const Select: FC<Props> = ({
+  setSelectedValue,
+  options,
+  defaultValue,
+}) => {
+  const onSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const value = event.target.value;
+
+    setSelectedValue(value || null);
+  };
+
+  return (
+    <select defaultValue="" onChange={onSelect} className={styles.select}>
```

Comment:
Can do it like this?

```suggestion
    <select
      defaultValue={defaultValue !== undefined && defaultValue !== null ? String(defaultValue) : ""}
      onChange={onSelect}
      className={styles.select}>
```